docs/demo/sentiment.rst

```diff
@@ -0,0 +1,73 @@
+Sentiment Analysis
+==================
+
+Introduction
+------------
+
+Sentiment Analysis predicts whether an input is positive or negative. The model is based on BERT base, and are trained on the binary classification setting of the Stanford Sentiment Treebank. It achieves about 93.4% accuracy on the dev set.
+
+Demo
+----
+
+You can either input a sentence into the textbox or select one sample from the select control.
```

Comment:
Currently we're giving an existence proof of "it's possible to have some web application to do sentiment analysis". But to be really useful to the end users, we need to teach them how to build it (not only show it's possible).

To make things simple, such instructions should be based on https://aws.amazon.com/serverless/ and https://aws.amazon.com/sagemaker/